tests/data/fits_region.fits

```diff
@@ -0,0 +1 @@
�b�                        @����e�                        ROTBOX @�]ֻ���@�x��!�                @<e*b�D       @�                             @�!                             ellipse@r      @�                      @uz���       @uP                             @u�                             ROTBOX @L      @S�                     @P@            @uP                             @u�                             ANNULUS@L      @S�                                    @uP                             @u�                             point                                                 @uP                             @u�                             point                                                 ?�      @       @      @      @      @      @      @       polygon                                               @$                              @                              BOX    @$      @4                                     	                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
```

Comment:
You should add a .gitattributes file to declare this as a binary. Copy the one from meas_algorithms.

Reply:
can you reorganize the commit history so that the .gitattributes file turns up before the fits file is committed?